data/Dockerfiles/letsencrypt/root/etc/dehydrated/config

```diff
@@ -0,0 +1,7 @@
+CONTACT_EMAIL="${LE_CONTACT_EMAIL:-$CONTACT_EMAIL}"
```

Comment:
Does this file not belong in the data/conf folder? - ` data/conf/letsencrypt/config`